public/js/pimcore/asset/image.js

```diff
@@ -427,30 +444,48 @@ pimcore.asset.image = Class.create(pimcore.asset.asset, {
 
         var markerDD = new Ext.dd.DD(marker);
 
-        this.marker = marker;
-    },
+        Ext.getCmp('remove_focal_point_' + this.id).setVisible(true);
+        Ext.getCmp('add_focal_point_' + this.id).setVisible(false);
 
-    getSaveData : function ($super, only) {
-        var parameters = $super(only);
+        this.marker = marker;
 
+    },
+    removeFocalPoint: function () {
         if(this["marker"]) {
+            this.marker.remove();
+            this["marker"] = false;
+            Ext.getCmp('remove_focal_point_' + this.id).setVisible(false);
+            Ext.getCmp('add_focal_point_' + this.id).setVisible(true);
+        }
+    },
+    getFocalPointCoordinates: function () {
```

Comment:
maybe change the function name to something not `get` as it is not returning the coordinates.

Reply:
thank you for the feedback, changed to set on https://github.com/pimcore/admin-ui-classic-bundle/pull/331/commits/b95886e193b0f31069fbe7973d190047adf88374

Reply:
oh wait, maybe something like "calculateFocalPointCoordinates" because it's not a setter either?